flexmeasures/ui/templates/crud/asset.html

```diff
@@ -369,6 +483,438 @@ <h3>All child assets for {{ asset.name }}</h3>
+                    const sensorData = await getSensor(sensor);
+                    const Asset = await getAsset(sensorData.generic_asset_id);
+                    const Account = await getAccount(Asset.account_id);
+                    sensorsUnits.push(sensorData.unit);
+            
+                    return `
+                    <div class="p-1 mb-3 border-bottom border-secondary">
+                        <span>
+                            <b>ID:</b> <a href="${apiBasePath}/sensors/${sensorData.id}">${sensorData.id}</a>, 
+                            <b>Unit:</b> ${sensorData.unit},
+                            <b>Name:</b> ${sensorData.name},
+                            <i class="fa fa-times" 
+                                onclick="removeSensorFromGraph(${index}, ${sensorIndex})"
+                                data-bs-toggle="tooltip" 
+                                data-bs-placement="top" 
+                                title="Remove Sensor"
+                                style="cursor: pointer;"
+                            >
+                            </i>
+
+                            <div style="padding-top: 1px;"></div>
+
+                            <b>Asset:</b> ${Asset.name},
+                            <b>Account:</b> ${Account?.name ? Account.name : "PUBLIC"}
+                        </span>
+                    </div>
+                    `;
+                })
+            );
+
+            const uniqueUnits = [...new Set(sensorsUnits)];
+
+            col.innerHTML = `
+                <div class="card m-0 p-1">
+                    <div class="card-body card-highlight ${newItem.title === selectedGraphTitle ? "border-on-click" : ""}" id="graph_${index}" onclick="selectGraph(${index})">
+                        <div class="d-flex align-items-center">
+                            <div>
+                                ${editingIndex === index
+                                    ? `<input type="text" class="form-control mb-2 me-2" id="editTitle_${index}" value="${newItem.title}" onkeydown="handleEnterKeyEvent(event, ${index})" />`
+                                    : `<h5 class="card-title me-2">${newItem.title}</h5>`
+                                }
+                            </div>
+
+                            <div>
+                                ${editingIndex === index
+                                    ? `<button class="btn btn-success btn-sm ms-2" onclick="saveGraphTitle(${index})">Save</button>`
+                                    : `<button class="btn btn-warning btn-sm ms-2" onclick="editGraphTitle(${index})">Edit</button>`
+                                }
+                            </div>
+                        </div>
+                        <h5 class="card-title pt-2"><b> Sensors: </b></h5>
+                        <div>
+                            ${sensorsContent.length > 0 ? sensorsContent.join("") : "<i>Add sensors on the right side</i>"}
```

Comment:
As without sensors, the graph is not shown, this should also be rendered in yellow (as the warning about differing units)